File: daily-programmer/001-easy-input.hs
import Text.Printf

import Utils

main :: IO ()
main = getInput >>= output
    where output s = putStr s >> appendFile "tmp/log.txt" s

getInput :: IO String
getInput = formMsg <$> input "your name > "
                   <*> input "your age > "
                   <*> input "your username > "

formMsg :: String -> String -> String -> String
formMsg = printf "your name is %s, \
                 \you are %s years old, \
                 \and your username is %s\n"

File: daily-programmer/001-easy-input.py
name = input('your name > ')
age = input('your age > ')
username = input('your username > ')

output = f'your name is {name}, you are {age} years old, and your username is {username}'

print(output)

with open('tmp/log.txt', 'a') as log:
    log.write(f'{output}\n')
